src/quadratureMethods/momentAdvection/velocity/VikasQuaziSecondOrderVelocityAdvection/VikasQuaziSecondOrderVelocityAdvection.C

```diff
@@ -105,17 +105,17 @@ Foam::velocityAdvection::VikasQuaziSecondOrder::VikasQuaziSecondOrder
     }
 
     {
-        IStringStream upwind("upwind");
-        IStringStream Minmod("Minmod");
-        weightOwnScheme_ = fvc::scheme<scalar>(own_, Minmod);
-        UOwnScheme_ = fvc::scheme<vector>(own_, upwind);
+        IStringStream weightLimiter("Minmod");
+        IStringStream ULimiter("upwind");
+        weightOwnScheme_ = fvc::scheme<scalar>(own_, weightLimiter);
+        UOwnScheme_ = fvc::scheme<vector>(own_, ULimiter);
     }
 
     {
-        IStringStream upwind("upwind");
-        IStringStream Minmod("Minmod");
-        weightNeiScheme_ = fvc::scheme<scalar>(nei_, Minmod);
-        UNeiScheme_ = fvc::scheme<vector>(nei_, upwind);
+        IStringStream weightLimiter("Minmod");
+        IStringStream ULimiter("upwind");
```

Comment:
Since you call the limiter for the abscissa "abscissaLimiter", I would suggest to keep the same name here because U is the abscissa. Alternative: velocityAbscissaLimiter, if you want to avoid the ambiguity.

Reply:
I also changed UOwn/NeiScheme to velocityAbscissaeOwn/NeiScheme